src/server/save_and_load.R

```diff
@@ -32,11 +32,25 @@ writeFilesForDigest <- function(workingSet, spectrumFilesState, surveyAndProgram
             write.csv(surveyAndProgramData$program_data_human_readable(), file = path, row.names = FALSE, na = "")
         })
     }
-    if (!is.null(modelRunState$spectrum_outputs)) {
+    if (!is.null(modelRunState$optim)) {
         dir.create("model_outputs")
-        paths <- doAndRememberPath(paths, file.path("model_outputs", glue::glue("spectrum_output.csv")), function(path) {
-            write.csv(modelRunState$spectrum_outputs, file = path, row.names = FALSE)
+        paths <- doAndRememberPath(paths, file.path("model_outputs", glue::glue("par.rds")), function(path) {
```

Comment:
use tempfile to avoid mutli-user corruption

Reply:
probably create a temp directory and then can put nicely named things in there

Reply:
just realised this is already happening, this all gets invoked inside a `withDir` on kine 82